sdk/modelparser/azure-digitaltwins-parser/src/main/java/com/azure/digitaltwins/parser/implementation/codegen/JavaDeclaration.java

```diff
@@ -95,9 +95,10 @@ public boolean getInheritDoc() {
     /**
      * Gets the value of the name decorated with appropriate keywords.
      *
+     * @param typeParams Compiled list of all typed parameters for declaration.
      * @return The value of the name decorated with appropriate keywords.
      */
-    public String getDecoratedName() {
+    public String getDecoratedName(String typeParams) {
```

Comment:
is there no default value concept in Java? This way, we don't need to pass in null when calling this method.

Reply:
unfortunately there is no default values in java :(